doc/sphinx/plugins/group-owner/groupDestroy.rst

```diff
@@ -0,0 +1,30 @@
+=============
+groupDestroy
+=============
+
+Delete a group
+==============
+
+
+.. admonition:: usage
+   :class: cmdusage
+
+   --osh groupDestroy --group GROUP
+
+.. program:: groupDestroy
+
+
+.. option:: --group GROUP
+
+   Group name to delete
+
+.. option:: --no-confirm 
+
+   Skip group name confirmation, but blame yourself if you deleted the wrong group!
+
+
+This command is able to delete any group you're an owner of.
+Granted users to the sibling restricted command `groupDelete` can delete any group.
+
```

Comment:
Trim empty lines

Reply:
This is generated documentation. I'll patch the generating script instead.

Reply:
Done, repushing a squashed version, with still a different commit for the bunch of removed empty lines. A linter would be nice indeed.